mitodl/open-discussions
# Add popup to channel settings link

#### Pre-Flight checklist


#### What are the relevant tickets?
TBD

#### What's this PR do?
Changes the channel settings link to a popup

#### How should this be manually tested?
Click the channel link. A popup should show up. You should be able to click the link and go to the channel settings page. If you click outside the popup the popup should disappear and you should remain on the page.
